contracts/extensions/ERC721AOwnersExplicit.sol

```diff
@@ -13,18 +13,21 @@ abstract contract ERC721AOwnersExplicit is ERC721A {
      */
     function _setOwnersExplicit(uint256 quantity) internal {
         require(quantity != 0, 'quantity must be nonzero');
-        require(currentIndex != 0, 'no tokens minted yet');
+        require(_nextTokenId > 1, 'no tokens minted yet');
```

Comment:
Can be changed to `!=` to save tiny gas https://github.com/chiru-labs/ERC721A/pull/59. `_nextTokenId` won't go below 1 anyway.